app/ide-desktop/lib/dashboard/src/layouts/TopBar.tsx

```diff
@@ -79,17 +80,19 @@ export default function TopBar(props: TopBarProps) {
             isAssetPanelVisible={isAssetPanelVisible}
             setIsAssetPanelVisible={setIsAssetPanelVisible}
           />
-          <UserBar
-            supportsLocalBackend={supportsLocalBackend}
-            page={page}
-            setPage={setPage}
-            isHelpChatOpen={isHelpChatOpen}
-            setIsHelpChatOpen={setIsHelpChatOpen}
-            projectAsset={projectAsset}
-            setProjectAsset={setProjectAsset}
-            doRemoveSelf={doRemoveSelf}
-            onSignOut={onSignOut}
-          />
+          {supportsCloudBackend && (
+            <UserBar
+              supportsLocalBackend={supportsLocalBackend}
```

Comment:
It seems like `supportsLocalBackend` is implicitly true here. In this case, is it necessary to pass it to the `UserBar` at all?

Reply:
Ah, my mistake!